### ISSUE TITLE
CJActionSheetPicker is not in NativeModules

### ISSUE THREAD

**Issue by zarcode** [4Catalyzer/react-native-actionsheet-picker#6]
Hi, need some help with 3. and 4. step of setting up
first I can't find this file libCJActionSheetPicker.a, I have this https://www.screencast.com/t/foq3wt9ExjSv

and when I add that file `CJActionSheetPicker` is not in `NativeModules` when I do require

**Comment by igorarkhipenko** [comment]
@zarcode met the same issue right there. Did you find out any workaround?

**Comment by zarcode** [comment]
@igorarkhipenko No, I switched to https://github.com/rnkit/rnkit-actionsheet-picker

**Comment by igorarkhipenko** [comment]
@zarcode thank you for such a fast response 👍 
Definitely will give it a try

**Comment by igorarkhipenko** [comment]
@zarcode I'm sorry for pinging you again, but is there any possibility of making fully independent pickers? I mean binding data refreshing when the first picker's value has been changed.

**Comment by zarcode** [comment]
@igorarkhipenko maybe setting defaultSelected value again will make component rerender (reset), not sure, haven't tried.
